web/tests/functional/detection_status/test_detection_status.py

```diff
@@ -52,6 +52,11 @@ def setUp(self):
             run_id = max(map(lambda run: run.runId, runs))
             self._cc_client.removeRun(run_id, None)
 
+        # Remove .clang-tidy configuration file if exists.
```

Comment:
If the `self._test_dir` is created temporary and initialized as an empty directory when each test set starts how is it possible that a .clang-tidy file is there when it is generated by one of the test in the test set.
If only one tests requires that test file, probably that one test should create it and remove it after it is finished.
Is that tidy config file required by multiple tests?

It is a bit interesting for me that the setUp starts with some cleaning up.